diff -u -r a/hw/input/virtio-input-hid.c b/hw/input/virtio-input-hid.c
--- a/hw/input/virtio-input-hid.c	2021-08-25 02:35:40.000000000 +0900
+++ b/hw/input/virtio-input-hid.c	2021-09-13 16:51:25.049082109 +0900
@@ -458,7 +458,7 @@
         .size      = 2,
         .u.bitmap  = {
             0,
-            (1 << (REL_WHEEL - 8))
+            0
         },
     },{
         .select    = VIRTIO_INPUT_CFG_ABS_INFO,
@@ -477,7 +477,7 @@
 };
 
 static Property virtio_tablet_properties[] = {
-    DEFINE_PROP_BOOL("wheel-axis", VirtIOInputHID, wheel_axis, true),
+    DEFINE_PROP_BOOL("wheel-axis", VirtIOInputHID, wheel_axis, false),
     DEFINE_PROP_END_OF_LIST(),
 };
 
